scalatest/src/main/scala/org/scalatest/tools/ScalaTestFramework.scala

```diff
@@ -430,7 +433,7 @@ Tags to include and exclude: -n "CheckinTests FunctionalTests" -l "SlowTests Net
               suiteClass.newInstance.asInstanceOf[Suite]
             else {
               val suiteClazz = wrapWithAnnotation.value
-              val constructorList = suiteClazz.getDeclaredConstructors()
+              val constructorList : Array[Constructor[_]] = suiteClazz.getDeclaredConstructors().map(_.nn)
```

Comment:
**Java method returns array**
Java method returns array of type `Array[Null | T]`. We need to map it to `Array[T]`.